plugins/plugin-chart-table/src/utils/formatValue.ts

```diff
@@ -0,0 +1,57 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+import { FilterXSS, getDefaultWhiteList } from 'xss';
```

Comment:
@kristw is this the recommended approach for sanitizing versus [Interweave](https://github.com/milesj/interweave)?

Reply:
Interweave looks interesting, but I don't think I need all its functionalities except for XSS production.

This library replaced `dompurify` in previous iteration of the new table chart and was responsible for a good mount of the 70% reduction on rendering time.

https://www.measurethat.net/Benchmarks/Show/7485/4/dompurify-vs-js-xss

Reply:
We did consider `interweave` during the last time we switch, but `xss` perform really well, being lightweight and independent from react. We also need to use output string (if I remember correctly) which was more difficult to do with `interweave`. 

This part of the logic is one of the thing that cost the most rendering time so any ms we can save is gold.